opennms-doc/guide-admin/src/asciidoc/text/kafka-forwarder/opennms-kafka-forwarder.adoc

```diff
@@ -0,0 +1,55 @@
+// Allow GitHub image rendering
```

Comment:
Thanks for adding the docs.

We also need to include this file somewhere for it to be part of the generated HTML.